asv_bench/benchmarks/tslibs/timedelta.py

```diff
@@ -10,6 +10,11 @@
 
 
 class TimedeltaConstructor:
+    def setup(self):
+        self.nptimedelta64 = np.timedelta64(3600)
+        self.dttimedelta = datetime.timedelta(seconds=3600)
+        self.td = Timedelta(3600, unit="s")
```

Comment:
Since we are here, let's add a benchmark for making Timedelta out of Timedelta, and I think we should also tweak existing tests. Currently, we benchmark both our method and `np.timedelta64` and `datetime.timedelta` at the same time. We should move the initialization into `setup`, in my opinion.